sympy/printing/latex.py

```diff
@@ -2325,6 +2322,9 @@ def latex(expr, fold_frac_powers=False, fold_func_brackets=False,
         Mul objects. Setting order to ``old`` uses the compatibility ordering
         for Add defined in Printer. For very large expressions, set the
         ``order`` keyword to ``none`` if speed is a concern.
+    root_notation : boolean, optional
```

Comment:
I think this is ok, but the order in the argument list should not be changed, should it, or else passing-by-position will fail.

Reply:
I see your point. However, is passing by positioning realistic when there are 15 keyword arguments? In the case one should move the mentioning in the documentation last as well, right? (The, sort of, annoying thing is that all the other keyword arguments are in alphabetical order...)